### PR TITLE
NAS-103089: Add help text to new "Region" field

### PR DESCRIPTION
**Pull request by Mrt134**
- Yarn build test: no issues.

### CODE REVIEW

**Comment by themylogin** [comment]
@Mrt134 us-east-1 default is not correct! It should be something like "Leave empty to detect region automatically. This won't work with private regions such as AWS GovCloud"

**Comment by Mrt134** [comment]
Understood, I'm just looking at your comments now and should have a PR update soon.

File: src/app/helptext/system/cloudcredentials.ts [hunk 1]
```diff
@@ -57,9 +57,14 @@ export const helptext_system_cloudcredentials = {
...
     )
   },
+
   region_s3: {
     placeholder: T("Region"),
-    tooltip: T(""),
+    tooltip: T('<a href="https://docs.aws.amazon.com/general/latest/gr/rande-manage.html" \
+ target="_blank">AWS resources in a geographic area</a>. Leave empty to \
+ automatically detect the correct public region for the bucket. Enter a \
+ private region name to allow the credential to find Amazon buckets \
```
Comment (themylogin) [inline src/app/helptext/system/cloudcredentials.ts]: Credential does not find anything, can it be something like "Enter a private region name to allow using this credential with S3 buckets created in that region."

Comment (Mrt134) [reply src/app/helptext/system/cloudcredentials.ts]: Yes, I can reword that.

**Comment by Mrt134** [comment]
Ok, I reworded the private region description slightly and added a link and example of configuring with AWS GovCloud

**Pull request merged**